test-data/unit/check-typeddict.test

```diff
@@ -1338,6 +1339,7 @@ reveal_type(m['director']['name']) # E: Revealed type is 'builtins.str'
 [out]
 
 [case testSubclassOfRecursiveTypedDict]
+# flags: --no-new-semantic-analyzer
```

Comment:
Similar to above.

Reply:
Opened https://github.com/python/mypy/issues/6445 for both (since this is essentially the same issue).